<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="dataTableWidget">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>137</y>
      <width>731</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="databaseNameLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="lastButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>37</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Last</string>
    </property>
   </widget>
   <widget class="QToolButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>36</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QToolButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>55</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QToolButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Edite</string>
    </property>
   </widget>
   <widget class="QToolButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>39</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QToolButton" name="firstButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>38</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>First</string>
    </property>
   </widget>
   <widget class="QToolButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>58</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QToolButton" name="infoButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>View info</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionGoClients"/>
    <addaction name="actionGoCampaigns"/>
    <addaction name="actionGoAdvertisements"/>
    <addaction name="actionGoPlatforms"/>
    <addaction name="actionGoSegments"/>
    <addaction name="actionGoUsers"/>
   </widget>
   <widget class="QMenu" name="menuQuery_Editor">
    <property name="title">
     <string>Query Editor</string>
    </property>
    <addaction name="actionOpenQuery"/>
   </widget>
   <widget class="QMenu" name="menuEdit_Form">
    <property name="title">
     <string>Edit Form</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionDelete"/>
    <addaction name="actionUpdate"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCampaignAndAdvertisements"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuQuery_Editor"/>
   <addaction name="menuEdit_Form"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGoClients">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clients</string>
   </property>
  </action>
  <action name="actionGoCampaigns">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Campaigns</string>
   </property>
  </action>
  <action name="actionGoAdvertisements">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Advertisements</string>
   </property>
  </action>
  <action name="actionOpenQuery">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionCampaignAndAdvertisements">
   <property name="text">
    <string>Campaign and Advertisements</string>
   </property>
  </action>
  <action name="actionGoPlatforms">
   <property name="text">
    <string>Media Platforms</string>
   </property>
  </action>
  <action name="actionGoSegments">
   <property name="text">
    <string>Audience Segments</string>
   </property>
  </action>
  <action name="actionGoUsers">
   <property name="text">
    <string>Users</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
